File: app/src/main/java/com/shevelev/comics_viewer/common/rhea/RheaOperationRunResult.java
package com.shevelev.comics_viewer.common.rhea;

/**
 * Result of operation runing
 */
public enum RheaOperationRunResult
{
    /**
     * Operation ran
     */
    Success,

    /**
     * Operation with such tag already ran
     */
    TagAlreadyRun,

    /**
     * Activity is not found
     */
    InvalidActivity
}

File: app/src/main/java/com/shevelev/comics_viewer/common/rhea/RheaOperationRunResult.kt
package com.shevelev.comics_viewer.common.rhea

/**
 * Result of operation runing
 */
enum class RheaOperationRunResult {
    /**
     * Operation ran
     */
    Success,
    /**
     * Operation with such tag already ran
     */
    TagAlreadyRun,
    /**
     * Activity is not found
     */
    InvalidActivity
}